app/helpers/application_helper.rb

```diff
@@ -54,12 +54,14 @@ def in_schools_namespace?
     request.path.start_with?('/schools')
   end
 
-  def summary_row(key, value, change_path = nil, id: nil, show_action: true)
+  def summary_row(key, value, change_path = nil, id: nil, show_action: true, display_as_list: false)
```

Comment:
Again - I don't think this needs to know how to format content - if a template wants it to show formatted content, than it should be passing it formatted content